static/js/settings_user_groups.js

```diff
@@ -234,7 +265,7 @@ exports.set_up = function () {
         var description = $(this).siblings('.description').text().trim();
         var name = $(this).siblings('.name').text().trim();
         var btn = $(this);
-
+        if (user_groups.is_member_of(group_id, people.my_current_user_id()) || me.is_admin) {
```

Comment:
The indentation here is off.

Reply:
@showell I've updated the PR, you may check!